### PR TITLE
Obs adaptor aux variables

### PR DESCRIPTION
**Pull request by garciampred**

### CODE REVIEW

File: cads_adaptors/adaptors/cadsobs/adaptor.py [hunk 1]
```diff
@@ -7,6 +7,23 @@
 
 logger = logging.getLogger(__name__)
 
+possible_aux_fields = [
```
Comment (EddyCMWF) [inline cads_adaptors/adaptors/cadsobs/adaptor.py:10]: What is this used for, I don't see it referenced elsewhere in the code

File: cads_adaptors/adaptors/cadsobs/adaptor.py [hunk 2]
```diff
@@ -26,16 +43,24 @@ def retrieve(self, request):
         dataset_source = self.handle_sources_list(dataset_source)
         mapped_request["dataset_source"] = dataset_source
         mapped_request = self.adapt_parameters(mapped_request)
-        # Request parameters validation happens here, not sure about how to move this to
-        # validate method
+        # Get CDM lite variables as a dict with mandatory, optional and auxiliar
```
Comment (EddyCMWF) [inline cads_adaptors/adaptors/cadsobs/adaptor.py:46]:
```suggestion
        # Get CDM lite variables as a dict with mandatory, optional and auxiliary
```

File: cads_adaptors/adaptors/cadsobs/adaptor.py [hunk 3]
```diff
@@ -26,16 +43,24 @@ def retrieve(self, request):
...
+        # Auxiliar are the former uncertainty etc. variables.
```
Comment (EddyCMWF) [inline cads_adaptors/adaptors/cadsobs/adaptor.py:47]:
```suggestion
        # Auxiliary variables are, generally, statistics associated with the mandatory and/or optional variables.
        # They are identified by matches with the full variable name, e.g. VAR_standard_deviation, 
        # where standard_deviation is the auxiliary variable.
        # The will be included as additional columns in the output file.
```

**Review by EddyCMWF — APPROVED** [review]

File: cads_adaptors/adaptors/cadsobs/adaptor.py (hunk 1 above)
Comment (garciampred) [reply cads_adaptors/adaptors/cadsobs/adaptor.py]: You are right, I removed it.

**Pull request merged**